narwhal/worker/src/handlers.rs

```diff
@@ -172,21 +172,21 @@ impl PrimaryToWorker for PrimaryReceiverHandler {
         let network = request
             .extensions()
             .get::<anemo::NetworkRef>()
-            .unwrap()
+            .ok_or_else(|| {
+                anemo::rpc::Status::internal("Unable to access network to send child RPCs")
+            })?
             .upgrade()
             .ok_or_else(|| {
                 anemo::rpc::Status::internal("Unable to access network to send child RPCs")
             })?;
```

Comment:
small nit but this could be simplified:
```suggestion
            .and_then(NetworkRef::upgrade)
            .ok_or_else(|| {
                anemo::rpc::Status::internal("Unable to access network to send child RPCs")
            })?;
```

Reply:
thanks!